problems/2.functions.js

```diff
@@ -2,15 +2,18 @@
 * @param {string} name - The name of the user to greet
 * @returns {null}
 */
-function logGreeting() {
-
-}
+function logGreeting(Avery) {
+  if (Hello)
+  return ("Avery");
```

Comment:
This just returns the string "Avery" we need to make this work for all names that are passed into our function. We don't want to hardcode "avery"